pom.xml

```diff
@@ -46,16 +46,17 @@
 
     <groupId>fish.payara.extras</groupId>
     <artifactId>payara-upgrade-tool</artifactId>
-    <version>2.0-SNAPSHOT</version>
```

Comment:
I wouldn't set it to 2.0 yet - we're liable to continue making changes

Reply:
done